src/main/java/org/mule/extension/http/api/request/HttpConnectivityValidator.java

```diff
@@ -130,6 +131,24 @@ private void validateResult(HttpRequest request, Result result) {
     getResponseValidator().validate(result, request);
   }
 
+  //This method is used for composer purposes only
+  public void validate(HttpExtensionClient client, HttpConstants.Protocol protocol, String host, Integer port)
+      throws ExecutionException, InterruptedException, ResponseValidatorTypedException {
+    String pathWithUriParams = replaceUriParams(requestPath, requestBuilder.getRequestUriParams());
+    String uriString = protocol.getScheme() + "://" + host + ":" + port + pathWithUriParams;
+    responseValidator = new SuccessStatusCodeValidator("200..299");
```

Comment:
Here it's replacing the old responseValidator. It would change the behaviour of this validator for future validate() calls, and that doesn't sound like something that validate() should do